{{cookiecutter.project_slug}}/clients/mobile/react-native/src/components/styled.tsx

```diff
@@ -1,22 +1,9 @@
-import { styled } from 'nativewind'
-import { KeyboardAvoidingView, ScrollView } from 'react-native'
+import { cssInterop } from 'nativewind'
+import type { FC } from 'react'
 import { Bounceable } from 'rn-bounceable'
 
-export const ScrollViewWind = styled(ScrollView, {
-  props: {
-    contentContainerStyle: true,
-  },
+cssInterop(Bounceable, {
```

Comment:
New way of doing NativeWind components.
We register the component, mark the prop that's going to be passthrough and then for Typescript purposes we cast the component to show up the new prop.

For native components (such as FlatList, ScrollView) these props are already taken cared of so you can find `contentContainerClassName` in `ScrollView` without the need of declaring a `ScrollViewWind` how we were doing previously